orm/src/test/java/org/hibernate/search/test/SearchInitializationTestBase.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.hibernate.search.test.util.ImmutableTestConfiguration;
+import org.hibernate.search.test.util.TestConfiguration;
+import org.junit.After;
+
+/**
+ * A base class for tests of the initialization process.
+ *
+ * <p>The main difference with {@link SearchTestBase} is that the test itself does not implement
+ * {@link TestConfiguration}, which enables using different configurations for different test methods.
+ * <p>When subclassing this class, Hibernate Search's initialization must be triggered manually by calling
```

Comment:
If we end up amending this commit, I don't think the `'s` is in order here. No need to fix it if we don't amend the commit though.